### DISCUSSION TITLE
"FP16 is not supported on CPU; using FP32 instead")

### DISCUSSION THREAD

**Discussion by kip-pixel** [openai/whisper#2342] [Q&A]
I'm getting this message warning anytime, i want to transcribe an audion, it does transcribe, but takes long long time to escute, kindly help guys, what am I missing? below is the msg.

/usr/local/lib/python3.10/dist-packages/whisper/__init__.py:146: FutureWarning: You are using `torch.load` with `weights_only=False` (the current default value), which uses the default pickle module implicitly. It is possible to construct malicious pickle data which will execute arbitrary code during unpickling (See https://github.com/pytorch/pytorch/blob/main/SECURITY.md#untrusted-models for more details). In a future release, the default value for `weights_only` will be flipped to `True`. This limits the functions that could be executed during unpickling. Arbitrary objects will no longer be allowed to be loaded via this mode unless they are explicitly allowlisted by the user via `torch.serialization.add_safe_globals`. We recommend you start setting `weights_only=True` for any use case where you don't have full control of the loaded file. Please open an issue on GitHub for any issues related to this experimental feature.
  checkpoint = torch.load(fp, map_location=device)
/usr/local/lib/python3.10/dist-packages/whisper/transcribe.py:126: UserWarning: FP16 is not supported on CPU; using FP32 instead
  warnings.warn("FP16 is not supported on CPU; using FP32 instead")

**Comment by ramy-nabil** [comment]
If you are using CPU then add "fp16=False" in the options.
